packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -192,6 +192,10 @@ private void applySettings(Map<String, Object> settings) {
 
           webView.setWebViewClient(webViewClient);
           break;
+        case "debuggingEnabled":
+          final boolean debuggingEnabled = (boolean) settings.get(key);
+
+          webView.setWebContentsDebuggingEnabled(debuggingEnabled);
```

Comment:
Is there an equivalent on iOS?

Reply:
debugging is enabled by default in dev build on iOS